src/backend/matrix_client.py

```diff
@@ -177,6 +177,7 @@ def __init__(self,
         self.profile_task:       Optional[asyncio.Future] = None
         self.server_config_task: Optional[asyncio.Future] = None
         self.sync_task:          Optional[asyncio.Future] = None
+        self.start_task:         Optional[asyncio.Future] = None
```

Comment:
This property is needed, because when the [`logout()`](https://github.com/mirukana/mirage/pull/80/commits/f325f6c528714a69ac40392d92ca506a27b1c297#diff-57d18905acc41568d85f243decc8d4beR140) is called, `_start()` is still a pending future task.